src/controller/model/endpoint.ts

```diff
@@ -633,6 +633,7 @@ class Endpoint extends Entity {
                     const attribute = cluster.getAttribute(item.attribute);
                     dataType = attribute.type;
                     attrId = attribute.ID;
+                    if (attribute.hasOwnProperty('manufacturerCode')) options.manufacturerCode = attribute.manufacturerCode;
```

Comment:
Could you try if just `options.manufacturerCode = attribute.manufacturerCode;` also works here?

Reply:
That would override the one from https://github.com/Koenkk/zigbee-herdsman/blob/9d122f3a531f6487f4e7f3b737ebf70549ded5c0/src/controller/model/endpoint.ts#L623 by undefined if the attribute doesn't have one set